specs/File Classification Spec.md

```diff
@@ -0,0 +1,95 @@
+# **PowerToys File Classification Spec**
+
+![MS PowerToys](https://hothardware.com/ContentImages/NewsItem/48038/content/Microsoft_PowerToys.jpg "PowerToys")
+## Users can quickly rename and group files.
+### Authors: Benjamin Leverette and Prudence Phillips
+### Spec Status: Draft
+## Overview
+
+### Elevator Pitch / Narrative 
+
+David has recently uploaded thousands of pictures from his camera to his Surface Laptop.  Unfortunately, the images all have generic names such as 'IMG_141' instead of labels that he can identify.  David downloads the File Classification PowerToy that allows him to create new folders with files that share a label, rename a group of files with a label, or change the name or portion of a name in a group of files to another name.
+
+### Customers
+
+Like all utilities from PowerToys, the File Classification feature is for power users and developers who are looking to tune and streamline their Windows experience for greater productivity.
+  
+### Problem Statement and Supporting Customer Insights
+
+Power users need a better way to organize files, from renaming files to creating new folders for similar files.  Our PowerToys Consumer Survey received feedback validating the usefulness of a feature that provides such functionality.
+
+### Existing Solutions or Expectations
+
+Users currently have to highlight a group of files and right click to rename them.  Users must manually create a folder, move files to that folder, and rename them.  There are third-party resources that allow users to rename files similarly.
+
+We expect users to install and enable PowerToys for Windows in order to access the File Classification utility.
+
+### Goals/Non-Goals
+
+Design and develop a feature that can rename and group files within an 8 week period.
+
+## Definition of Success
+
+### Expected Impact: Customer, and Technology Outcomes, Experiments + Measures
+
+Our PowerToys Consumer Survey received an abundant amount of participation and feedback from a community of passionate power users.  This feature will give them the ability to rename, group and organize files in a way that makes their virtual library of files more organized and efficient.  As interns, we would have 8 weeks to complete the project.
+
+## Requirements
+
+### Functional Requirements
+
+**1. Initial UX/UI**
```

Comment:
Change this and all other headings with: `** X Y Z **` to: 
`#### X Y Z`
As the hashtags indicate a header, while the asterisks just indicate a bold.